### PR TITLE
docs: modify the docs directory 

### PR DESCRIPTION
**Pull request by guoqqqi**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

Changed the directory structure under docs.

### CODE REVIEW

**Comment by guoqqqi** [comment]
Waiting to fix broken links and images.

File: docs/zh/latest/plugins/proxy-rewrite.md [hunk 1]
```diff
@@ -17,9 +17,13 @@
 #
 -->
 
+<<<<<<< HEAD:docs/zh/latest/plugins/proxy-rewrite.md
+# proxy-rewrite
+=======
 - [English](../../plugins/proxy-rewrite.md)
 
 # 目录
+>>>>>>> master:doc/zh-cn/plugins/proxy-rewrite.md
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/proxy-rewrite.md:26]: Conflict

File: docs/zh/latest/config.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "version": 2.3,
+  "sidebar": {}
```
Comment (juzhiyuan) [inline docs/zh/latest/config.json:3]: We could fill this sidebar in another PR when this one gets merged.

File: docs/en/latest/config.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "version": 2.3,
+  "sidebar": {}
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:3]: we could fill this sidebar in another PR, not urgent currently.

**Comment by juzhiyuan** [comment]
https://github.com/apache/apisix/pull/3661/checks?check_run_id=1975897061#step:14:733

cc @spacewander to take a look at this CI

**Review by juzhiyuan — APPROVED** [review]

**Review by Yiyiyimu — APPROVED** [review]

**Pull request merged**